README.md

```diff
@@ -61,21 +66,77 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
 - Spanish
 - Hungarian
 
-## Binaries supported right now
+## Installation
+
+### Debian
+
+Download the `heroic_x.x.x_amd64.deb` from the Releases section
+
+```bash
+sudo dpkg -i heroic_x.x.x_amd64.deb
+```
+
+### Arch (AUR)
+
+[![AUR version](https://img.shields.io/aur/version/heroic-games-launcher-bin?style=flat-square)](https://aur.archlinux.org/packages/heroic-games-launcher-bin/)
+
+AUR page: [https://aur.archlinux.org/packages/heroic-games-launcher-bin/](https://aur.archlinux.org/packages/heroic-games-launcher-bin/)
+
+To install it manually use:
+
+```bash
+git clone https://aur.archlinux.org/heroic-games-launcher-bin.git
+
+cd heroic-games-launcher-bin
+
+makepkg --cleanbuild --syncdeps --install --clean --rmdeps
+```
+
+Otherwise you can install it via your prefered AUR helper, e.g. yay.
+
+### Fedora
 
-- DEB, RPM and AppImage
-- Recommended AUR version: https://aur.archlinux.org/packages/heroic-games-launcher-bin/
-- Fedora (COPR): https://copr.fedorainfracloud.org/coprs/atim/heroic-games-launcher/
-- Might think about Flatpak And/Or SNAP in the future as well
+Heroic for Fedora is available on [the COPR repo](https://copr.fedorainfracloud.org/coprs/atim/heroic-games-launcher/) or as the binary on the releases page.
 
-## How to build and run locally
+### Other Distributions
 
-To be able to run you will need to have NodeJS installed locally and follow the instructions below:
+Download the `heroic-x.x.x.AppImage` from the Releases section.
 
-### Steps:
+To make it executable use:
 
-- Clone the Repository.
-- On the project folder run `yarn install`.
-- Run `yarn build` to build the React Assets.
-- Run `yarn start`.
-- To build the binaries run `yarn dist {package to create}`.
+```bash
+chmod +x heroic-x.x.x.AppImage
+```
+
+To run it use:
+
+```bash
+./heroic-x.x.x.AppImage
+```
+
+### Build, install and run locally
+
+Requires NodeJS to build
+
+```bash
+git clone https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher.git
+
+cd HeroicGamesLauncher
+
+yarn
+
+yarn start
+
+yarn electron
+
+yarn dist {package to create}
```

Comment:
This is pretty ambiguous, please change the headings to "Build binaries locally.".
Also stuff for macOS will be added soon.